docs/content/error/$compile/iscp.ngdoc

```diff
@@ -12,9 +12,11 @@ myModule.directive('directiveName', function factory() {
     scope: {
       'attrName': '@', // OK
       'attrName2': '=localName', // OK
+      'attrName2': '<?localName', // OK
```

Comment:
I think it's better to increment all attrName suffixes (e.g. here: _attrName2_  -->  _attrName3_).